libibverbs/init.c

```diff
@@ -667,18 +667,27 @@ int ibverbs_init(void)
 {
 	char *env_value;
 
-	if (getenv("RDMAV_FORK_SAFE") || getenv("IBV_FORK_SAFE"))
+	bool want_ibv_fork_init = false;
+	env_value = getenv("RDMAV_FORK_SAFE");
+	want_ibv_fork_init = env_value && (strcmp(env_value, "0") != 0);
+	if (!want_ibv_fork_init) {
+		env_value = getenv("IBV_FORK_SAFE");
+		want_ibv_fork_init = env_value && (strcmp(env_value, "0") != 0);
+	}
+	if (want_ibv_fork_init)
 		if (ibv_fork_init())
 			fprintf(stderr, PFX "Warning: fork()-safety requested "
 				"but init failed\n");
 
-	/* Backward compatibility for the mlx4 driver env */
-	env_value = getenv("MLX4_DEVICE_FATAL_CLEANUP");
+	env_value = getenv("RDMAV_ALLOW_DISASSOC_DESTROY");
```

Comment:
I think this part is from another commit?

Reply:
No, I am trying to make all the variables behave the same way.